src/EnumCaseAttributesTrait.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the "elao/enum" package.
+ *
+ * Copyright (C) Elao
+ *
+ * @author Elao <contact@elao.com>
+ */
+
+namespace Elao\Enum;
+
+use Elao\Enum\Attribute\EnumCase;
+
+trait EnumCaseAttributesTrait
+{
+    private function getEnumCaseAttribute(): ?EnumCase
```

Comment:
(methods are private, but can be used by any userland implementation using this trait, as not marked `@internal`)